Packs/CIRCL/Integrations/CirclCVESearch/CirclCVESearch_test.py

```diff
@@ -0,0 +1,136 @@
+import json
+import os
+from pathlib import Path
+from CirclCVESearch import (cve_command, valid_cve_id_format, Client, generate_indicator, parse_cpe)
+from CommonServerPython import DemistoException, argToList, EntityRelationship
+import pytest
+
+BASE_URL = 'https://cve.circl.lu/api/'
+
+
+def test_wrong_path():
+    bad_url = 'https://cve.bad_url'
+    client = Client(base_url=bad_url)
+
+    with pytest.raises(DemistoException) as excinfo:
+        cve_command(client, {"cve": 'cve-2000-1234'})
+
+    expected_exception_message = 'Verify that the server URL parameter is correct'
+    assert expected_exception_message in str(excinfo.value), 'Bad error response when bad URL is given'
+
+
+def test_bad_cve_id():
+    bad_cve_id = 'CVE-bad-cve'
+    client = Client(base_url=BASE_URL)
+
+    with pytest.raises(DemistoException) as excinfo:
+        cve_command(client, {'cve': bad_cve_id})
+
+    expected_exception_message = f'"{bad_cve_id}" is not a valid cve ID'
+    assert str(excinfo.value) == expected_exception_message
+
+
+def test_cve_id_validation():
```

Comment:
Why not using pytest.parametrize? You can see examples in the repo.

Reply:
Old code, not mine :\

Reply:
So why its part of your PR?